src/metabase/models/serialization.clj

```diff
@@ -981,6 +1218,7 @@
          (filter some?)
          (reduce set/union #{}))))
 
+;;; <h2>Memoizing appdb lookups</h2>
```

Comment:
I want Markdown here! 😁

Reply:
Oh, turns out it's ok to use markdown here... Hope you won't mind if I change headers to use `## `. :)